src/trumbowyg.js

```diff
@@ -1433,6 +1440,24 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
 
                     t.syncCode();
                     t.semanticCode(false, true);
+                    try {
+                        var listId = window.getSelection().focusNode.parentNode;
+                        var arrayKey = param + 'Class';
+
+                        var arr = t.o[arrayKey];
+                        if (arr) {
+                            $(listId).removeClass();
```

Comment:
Prefer just remove the class from the option, not *all* classes, since the user can add a class by other ways

Reply:
is done ;)

Reply:
I did not understand the purpose of this section.
This line removes all classes, then you remove again each class, and finally re-add them back.
We can give a string with all classes to addClass function, so this can be simplified.

Do not forget that users can add classes using other ways, so we do not want to clean them here. Just add the wanted classes.

Reply:
hello @Alex-D 
I fixed it by remove the class removing lines.

best regards
Emanuel